packages/cli/src/cmds/validator/keys.ts

```diff
@@ -185,7 +183,7 @@ export function resolveKeystorePaths(fileOrDirPath: string): string[] {
   if (fs.lstatSync(fileOrDirPath).isDirectory()) {
     return fs
       .readdirSync(fileOrDirPath)
-      .filter((file) => !depositDataPattern.test(file))
+      .filter((file) => isVotingKeystore(file))
```

Comment:
I think this may be too strict. We should accept any valid keystore in the directory and not skip over a keystore that doesn't have the standard file naming convention.

Reply:
I see your point. Will an approach that attempts to parse any file in the directory but recovers if it fails (instead of crashing the app) be preferably? 

The other option is to include yet another filter to skip files that matches known bad formats. Initial issue was with the deposit data file, now it is the password.txt file that cause the issue for this PR. The problem with this though is that the list of blacklisted pattern can theoretically continue to grow.

Reply:
Had an implementation which I forgot to push which does not introduce a new `KeystoreDecryptPool` [here](https://github.com/ChainSafe/lodestar/pull/4117/commits/b5e5f648ae3534134beda2a55734bdfb968c233c)

I'll look into https://github.com/ChainSafe/lodestar/issues/4166 and see how that would affects this.